resources/dicts/patrols/forest/hunting/newleaf.json

```diff
@@ -3483,7 +3483,7 @@
                 ]
             },
             {
-                "text": "It's a wonderful day, wandering through the sunshine kinda randomly around c_n's forest searching for the signs of bird nests and swarming up the trees for them. s_c is even able to show off a little, inching cautiously out onto the spindly branches of a willow to snag a hard-to-reach nest.",
+                "text": "It's a wonderful day, wandering through the sunshinekind of randomly around c_n's forest searching for the signs of bird nests and swarming up the trees for them. s_c is even able to show off a little, inching cautiously out onto the spindly branches of a willow to snag a hard-to-reach nest.",
```

Comment:
I noticed you're missing a space here between "sunshine" and "kind"

Reply:
Working on!